### DISCUSSION TITLE
Get GroupMembers by Group

### DISCUSSION THREAD

**Discussion by lchastain** [jsforce/jsforce#1485] [Ideas]
This endpoint is new as of v36 of the API:  .../services/data/v36.0/sobjects/Group/{groupId}/GroupMembers (Get all members of a specified group).  Using the jsforce client, it fails due to 'The requested resource does not exist'.  Looking into what jsforce is requesting, it has added a '/describe' to that endpoint and it is true that there is no such resource.  Not sure if this is an API problem as well, but I'm not trying to get the '/describe' and without it, the endpoint does respond properly when sent directly to the v36.0 api, outside of jsforce.

**Comment by stomita** [comment]
Which method did u call to invoke the endpoint?

**Comment by lchastain** [comment]
After having used the client to obtain a usable connection (conn), the method used was:
conn.sobject("Group/".concat(group.Id).concat("/GroupMembers")).find({})

**Comment by stomita** [comment]
You can't use the `sobject()` parameter other than the object name. If you want to call the REST endpoint directly, use `Connection#request(url)`.
